src_python/yaafelib/audiofeature.py

```diff
@@ -25,14 +25,32 @@
 import yaafelib.core as yaafecore
 
 
+# Taken from python package `six`
+# https://bitbucket.org/gutworth/six/src/2c12cd64ff0c7797bb30b0d466e902f7ecd6e562/six.py?fileviewer=file-view-default#six.py-826
+def add_metaclass(metaclass):
+    """Class decorator for creating a class with a metaclass."""
+    def wrapper(cls):
+        orig_vars = cls.__dict__.copy()
+        slots = orig_vars.get('__slots__')
+        if slots is not None:
+            if isinstance(slots, str):
+                slots = [slots]
+            for slots_var in slots:
+                orig_vars.pop(slots_var)
+        orig_vars.pop('__dict__', None)
+        orig_vars.pop('__weakref__', None)
+        return metaclass(cls.__name__, cls.__bases__, orig_vars)
+    return wrapper
+
+
 def check_dataflow_params(f):
     def _safe_get_dataflow(cls, params, samplerate):
         # filter parameters and set missing parameters with default values
-        for k, v in params.iteritems():
+        for k, v in params.items():
```

Comment:
Sure it's better to not depend on `six`, an alternative way is to create a [_compat.py](https://github.com/pallets/werkzeug/blob/master/werkzeug/_compat.py) in this repo.

You may need to resolve the dependency issue using python setuptools in https://github.com/Yaafe/Yaafe/issues/20 first, since Yaafe is already depending on `numpy`.